docs/source/cli_help/slide-out.rst

```diff
@@ -0,0 +1,59 @@
+.. role:: bash(code)
+
+.. _slide-out:
+
+slide-out
+---------
+Usage:
+
+.. code-block:: shell
+
+    git machete slide-out [-d|--down-fork-point=<down-fork-point-commit>] [-M|--merge] [-n|--no-edit-merge|--no-interactive-rebase] <branch> [<branch> [<branch> ...]]
+
+Removes the given branch (or multiple branches) from the branch tree definition.
+Then synchronizes the downstream (child) branches of the last specified branch on the top of the upstream (parent) branch of the first specified branch.
+Sync is performed either by rebase (default) or by merge (if :bash:`--merge` option passed).
+
+The most common use is to slide out a single branch whose upstream was a :bash:`develop/master` branch and that has been recently merged.
```

Comment:
Nit: 
```suggestion
The most common use is to slide out a single branch whose upstream was a ``develop``/``master`` branch and that has been recently merged.
```

Reply:
So that it doesn't look like a single branch of an usual `develop/master` name ;p